projects/1.16/assets/observable/observable/lang/zh_cn.json

```
@@ -0,0 +1,20 @@
+{
+  "screen.observable.profile": "性能剖析",
+  "screen.observable.results": "结果",
+  "key.observable.profile": "显示剖析界面",
+  "category.observable.keybinds": "Observable",
+  "text.observable.profile_tps": "TPS 性能剖析",
+  "text.observable.profile_fps": "FPS 性能剖析",
+  "text.observable.sampler": "Profile with sampler (adds overhead!)",
+  "text.observable.unimplemented": "暂未实现",
+  "text.observable.unavailable": "服务端没有安装 Observable 模组",
+  "text.observable.no_permissions": "你没有运行性能剖析的权限",
+  "text.observable.error": "执行错误： %s",
+  "text.observable.report": "请将这些提交至 github.com/tasgon/observable/issues",
+  "text.observable.entity_not_found": "在 %s 未发现实体。 实体已死亡或被移除",
```

```suggestion
  "text.observable.entity_not_found": "在 %s 未发现实体。实体已死亡或被移除",
```

中文标点符号前后不需要加空格。还有后面这句是疑问句，怎么变成陈述句了